pdns/dnsdist.hh

```diff
@@ -418,6 +419,7 @@ struct IDState
   uint16_t origID;                                            // 2
   uint16_t origFlags;                                         // 2
   int delayMsec;
+  uint32_t tempFailureTTL;
```

Comment:
It would be slightly better to move this field with the other `uint32_t` one, for alignment purpose.